frontend/src/pages/uid.tsx

```diff
@@ -0,0 +1,99 @@
+//Chakra Imports
+import {
+	Box,
+	Flex,
+	Text,
+} from "@chakra-ui/react";
+import { GetStaticPropsContext } from "next";
+import { NextSeo } from "next-seo";
+import Announcement from "src/components/site/Announcement";
+// Server side
+import { markdownSSR } from "src/mdx-helpers/ssr";
+import { MarkdownContent } from "src/mdx-helpers/types";
+//Styles
+import heroStyles from "../styles/Hero.module.css";
+
+const crypto: Crypto | undefined = self.crypto ?? (self as { msCrypto?: Crypto }).msCrypto; // eslint-disable-line no-restricted-globals
+
+const getRandomBytes: (n: number) => Uint8Array = (crypto ?
+  function () { // Browsers
+    return function (n: number): Uint8Array {
+      var a = new Uint8Array(n);
+      for (var i = 0; i < n; i += 65536) {
+        crypto.getRandomValues(a.subarray(i, i + Math.min(n - i, 65536)));
+      }
+      return a;
+    };
+  } :
+  function () { // Node
+    return require("crypto").randomBytes as (n: number) => Uint8Array;
```

Comment:
Disable eslint `@typescript-eslint/no-var-requires`.
```suggestion
    return require("crypto").randomBytes as (n: number) => Uint8Array; // eslint-disable-line @typescript-eslint/no-var-requires
```